src/Common/Core/Impl/IO/FileSystem.cs

```diff
@@ -8,6 +8,9 @@
 
 namespace Microsoft.Common.Core.IO {
     public sealed class FileSystem : IFileSystem {
+        private static FileSystem _instance = new FileSystem();
+        public static IFileSystem Current => _instance;
```

Comment:
Let's not introduce more singletons.
This class is lightweight and can be instantiated even when it isn't accessible from MEF